usl_models/usl_models/flood_ml/model.py

```diff
@@ -177,6 +178,27 @@ def train(
 
         return model_history
 
+    def load_model(self, filepath: str):
+        """Loads weights from an existing file.
+
+        Args:
+            filepath: Path to the weights file to load into the current model.
+        """
+        self._model.load_weights(filepath)
+        logging.info("Loaded model weights from %s", filepath)
+
+    def save_model(self, filepath: str, overwrite: bool = False):
```

Comment:
Need to add an explicit `-> None:` to allow MyPy to pay attention to the function.

Reply:
Done.